kong/plugins/prometheus/exporter.lua

```diff
@@ -146,6 +150,21 @@ local function collect()
                  "/metrics endpoint: ", err)
   end
 
+  -- upstream targets accessible?
+  local balancer = require "kong.runloop.balancer"
+  local upstreams_dict = balancer.get_all_upstreams()
+  for name, id in pairs(upstreams_dict) do
+      local health_info = balancer.get_upstream_health(id)
+      for target, status in pairts(health_info) do
```

Comment:
Typo

Reply:
Yep, that's why we need tests

Reply:
@Tieske that's why we need a type checker ;)

Reply:
existing linter set up would have caught this